.changeset/proud-queens-warn.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-backend-module-github': patch
+---
+
+- Make it possible to inject custom user and team transformers when configuring the `GitHubOrgEntityProvider`
```

Comment:
change `GitHub` to `Github` ONLY in class/type names too, here and in the other files. It's still (perhaps confusingly) GitHub with an uppercase H like the proper product name, in regular text.

Reply:
Anything that is a symbol in code should use `Github`. If there's text or documentation that's referring to the service then you should use "GitHub". It's what we've settled on, mainly because "GitHub" is the correct way to capitalize it, but camel casing it as `gitHubClient` looks very awkward and we want it to be consistent.